source/1_engine/rex_engine/include/rex_engine/filesystem/vfs.h

```diff
@@ -6,5 +6,5 @@ namespace rex
 }
 
 #ifdef REX_PLATFORM_WINDOWS
-#include "rex_engine/platform/win/filesystem/vfs.h"
+#include "rex_engine/platform/win/filesystem/win_vfs.h"
```

Comment:
no, I made it win_vfs to make it easier on intellisense and so you can see in the IDE which file you're editing, the windows specific version of the cross platform version

Reply:
This is different from the other platform specific code we are using, I would pick one and go with it. Please do not mix and match.

I can see it in my IDE which one I am editing if I look at the Solution Explorer (in VS) or the the file browser (in Code)

Reply:
these mistakes of not seeing which file you're operating in get more obvious when rex gets bigger. yes you can see it in solution explorer or in the browser, but those things can easily be overlooked.
also swapping between header and cpp file can get confusing if the files have the same name
the format we should be using is, if there's a cross platform filename for something, you add a prefix of the platform to the file in the platform specific code. if there's no cross platform equivalent, the platform prefix isn't needed.

Reply:
I don't understand why you suddenly change again, you did this for the paths but for the VFS you are reluctant again. Either change the path ( and everything else that uses this tactic to this strategy ) or use the design you used for the path on this thing as well but please don't mix and match it will get confusing very fast.

Reply:
it's possible I forgot something, I thought I changed path and vfs to be win_path and win_vfs, it's possible due to all the merges, some of that got lost